keps/sig-apps/1591-daemonset-surge/kep.yaml

```diff
@@ -27,7 +30,7 @@ feature-gates:
       - kube-controller-manager
 disable-supported: true
 stage: "alpha"
```

Comment:
This should read beta, right?

Reply:
So, this should be intended state or current state? I can change this to beta, if it has to be intended state.

Reply:
Yes, this should be the intended state.